packages/codemods/src/v5/keep-previous-data/__testfixtures__/default.input.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react'
```

Comment:
@TkDodo please review the examples.

Reply:
thanks for the comments, this looks perfect